requirements.txt

```diff
@@ -55,7 +55,9 @@ parso==0.8.3
 pexpect==4.8.0
 pickleshare==0.7.5
 Pillow==9.0.0
+Pint==0.18
 pluggy==1.0.0
+periodictable==1.6.0
```

Comment:
Just a note to our future selves: we're effectively importing constants from these repositories, and any changes to those constants (which are likely to be small, but nevertheless!) will change our results.

We should exercise caution when accepting updates to these modules, to the point where I'm almost considering adding a test so that we can detect any changes... I know that sounds mad, but these changes here have the potential to cause less than negligible variations in results.